lib/actions/api.js

```diff
@@ -355,7 +355,7 @@ export function routingQuery(searchId = null, updateSearchInReducer = false) {
  * @param  {Object} options  The options object for the API request
  * @return {string}          The URL to use for making the http request
  */
-function makeApiUrl(config, endpoint, options) {
+export function makeApiUrl(config, endpoint, options) {
```

Comment:
probably a question more for me, but why does this get exported?

Reply:
It's so we can use it in `default-map.tsx`!

Reply:
Great, thanks!